### DISCUSSION TITLE
at.ac.iiasa.ixmp.exceptions.IxException

### DISCUSSION THREAD

**Discussion by my0828** [iiasa/message_ix#728] [MESSAGE modeling]
Run error with command of `message_ix.macro.calibrate()`,How to solve it.
Errro message:
```File：D:\Anaconda\lib\site-packages\ixmp\backend\jdbc.py:580, in JDBCBackend.write_file(self, path, item_type, **kwargs)
    577         raise ValueError("write to GDX requires a Scenario object")
    579     # include_var_equ=False -> do not include variables/equations in GDX
--> 580     self.jindex[ts].toGDX(str(path.parent), path.name, False)
    581 elif path.suffix == ".csv" and item_type is ItemType.TS:
582     models = set(filters.pop("model"))
at.ac.iiasa.ixmp.exceptions.IxException: at.ac.iiasa.ixmp.exceptions.IxException: This Scenario must be checked in before performing this operation!```

**Comment by glatterf42** [comment]
Hi @my0828, this seems to be a follow-up discussion from #727. While I appreciate the effort to keep separate issues in separate discussions, we would need much the same information here as we did in the last one, so we could also continue resolving your errors there. With all errors, you can ensure to receive the help you want faster and more precise by being as accurate as you can be. An accurate error message includes: 

- which code you are trying to run (all relevant commands up to and including the one causing the error)
- which error message you receive (as complete as possible)
- which version of the code you are using (`message-ix show-versions` is our convenience tool for that)

By including this information, the community can understand quicker what exactly your problem is and how to reproduce it on our own systems. This helps us figuring out a solution that we can then share with you.
